apps/web/src/services/administration/store/user-page-store.ts

```diff
@@ -28,11 +28,17 @@ export const useUserPageStore = defineStore('user-page', {
         selectedUser: {} as UserModel | WorkspaceUserModel,
         totalCount: 0,
         selectedIndices: [],
+        // TODO: plan to organize it after completing the modal.
         modalVisible: {
+            form: false,
+            //
             status: false,
             create: false,
+            //
             update: false,
-            remove: false,
+            delete: false,
+            enable: false,
+            disable: false,
```

Comment:
surprised: 🤪 quite a lot of modals!

Reply:
I'm organizing because there are too many types, and having individual visibility might be more confusing!